Kudu.Core/Deployment/FetchDeploymentManager.cs

```diff
@@ -240,6 +258,15 @@ public class FetchDeploymentManager : IFetchDeploymentManager
                             innerLogger.Log(ex);
                         }
 
+                        if (updateStatusObj != null)
+                        {
+                            // Set deployment status as failure if exception is thrown
+                            updateStatusObj.DeploymentStatus = Constants.BuildFailed;
+                            updateStatusObj.DeploymentStatusInt = Constants.BuildFailed;

```

Comment:
How are these two different?

Reply:
I think DeploymentStatus is a leftover artifact from when there was confusion between OperationStatus and DeploymentStatus. @purva-vasudeo it looks like only DeploymentStatusInt is read by the front end, can we remove DeploymentStatus from this status object?